src/azure-cli/HISTORY.rst

```diff
@@ -313,7 +313,7 @@ Release History
 * Fix IcM179210086: unable to create custom metric alert for their Application Insights metric
 
 **NetAppFiles**
-
+* az volume create: Adds snapshot-id as a parameter to create volume this will allow users to create a volume from existing snapshot.
```

Comment:
Don't modify this file. We have a automation pipeline to update this based PR title and PR body content while realeasing.

Reply:
Thanks you for the clarification